lib/widgets/screens/inbox/channel_messages/channel_messages_screen.dart

```diff
@@ -66,16 +66,15 @@ class _ChannelMessagesScreenState extends State<ChannelMessagesScreen>
             final participant = channel.participants
                 .firstWhere((item) => item.user.id != _authenticatedUser.id)
                 .user;
-            final String channelName = participant.fullName!;
-            final String? avatarUrl = participant.avatarUrl;
             buildPageRouteScaffold((scaffoldModel) {
               scaffoldModel.setParams(
                 appBar: AppBarFactory.createChannelMessagesAppBar(
                   context: context,
-                  channelName: channelName,
+                  userFullName: participant.fullName!,
```

Comment:
I think it's better to use `?? "Unknown user"` here. There have been cases in the past of using `!` with fields that were unsupported by the mock server.